docs/reference/modules/asset.md

```diff
@@ -31,6 +32,38 @@ By default, `watch` is set to `true`. A truthy value will cause the application
 
 When the `APOS_UPLOADFS_ASSETS` environment variable is present, this optional property can be used to configure an `uploadfs` instance that differs from the one configured by the `attachment` module, allowing changes in where assets from the webpack build process are stored and how they are served. Full documentation for uploadfs can be found [here](https://www.npmjs.com/package/uploadfs).
 
+### `rebundleModules`
+
+The `rebundleModules` option allows for overridding the `bundles` properties passed into `webpack` at the individual module level, including modules add through npm. This option takes an object with module names, or module names with a suffix made up of a `:` and individual file name from the module `ui/src` folder, as properties. Each property takes the name of the bundle where the file should be added as a string.
+
+#### Example
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  options: {
+    rebundleModules: {
+      // Everything from the fancy-form module should go in the regular "main" bundle
+      'fancy-form': 'main',
+      // Everything from the basic-product module should go in the "secondary" bundle
+      'basic-product': 'secondary',
+      // The ui/src/form.js code from the @dcad/form module should be retargeted to the
```

Comment:
"Code originally designated as part of the `form` bundle from the `@dcad/form module"

Reply:
Changed (both files).